src/main/java/calculator/Splitter.java

```diff
@@ -0,0 +1,72 @@
+package calculator;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class Splitter {
+    private String[] arrays;
+    private List<Integer> integers;
+    private List<CalculatorSign> signs;
+
+    protected Splitter() {
+    }
+
+    public Splitter(String data) {
+        if (isBlank(data)) {
+            throw new IllegalArgumentException(String.format("입력 데이터가 올바르지 않습니다. : [%s] ", data));
+        }
+
+        this.arrays = data.split(" ");
+
+        if (!isSupportFormat()) {
+            throw new IllegalArgumentException(String.format("입력 데이터가 올바르지 않습니다. : [%s] ", data));
+        }
+
+        setIntegers();
+        setCalculatorSigns();
+    }
+
+    private void setCalculatorSigns() {
+
+        this.signs = IntStream.range(0, arrays.length)
+                .filter(i -> i % 2 == 1)
+                .mapToObj(i -> CalculatorSign.findByCodeName(arrays[i]))
+                .collect(Collectors.toList());
+
+    }
+
+    private void setIntegers() {
+        this.integers = IntStream.range(0, arrays.length)
+                .filter(i -> i % 2 == 0)
+                .mapToObj(i -> Integer.parseInt(arrays[i]))
+                .collect(Collectors.toList());
+    }
+
+    private boolean isSupportFormat() {
+        boolean isCalculatorSign = IntStream.range(0, this.arrays.length)
+                .filter(i -> i % 2 == 1)
```

Comment:
스트림 내부에서 사용하는 파라미터도 i 라는 네이밍 보다는 의미를 가지는 이름을 사용하는건 어떨까요 ? 
```suggestion
                .filter(i -> isOdd(i))
```
와 같이 메소드로 구분하여 가독성을 높여줄수 있을것 같아요 😄